eZ/Bundle/EzPublishCoreBundle/Resources/config/image.yml

```
@@ -83,6 +85,18 @@ services:
         tags:
             - { name: liip_imagine.cache.resolver, resolver: ezpublish }
 
+    ezpublish.image_alias.imagine.cache_resolver_decorator_factory:
+        class: '%ezpublish.image_alias.imagine.decorated_cache_resolver_factory.class%'
+        arguments:
+            - '@ezpublish.config.resolver.chain'
```

```suggestion
            - '@ezpublish.config.resolver'
```
Yes please, this is the one we should care about when it comes to BC.